src/panels/config/automation/ha-automation-picker.ts

```diff
@@ -1051,28 +1051,28 @@ class HaAutomationPicker extends SubscribeMixin(LitElement) {
     this._applyFilters();
   }
 
-  private _showInfo(ev) {
-    const automation = ev.currentTarget.parentElement.anchorElement.automation;
+  private _showInfo = (ev) => {
```

Comment:
...then type the parameter as `HaMenuItem` (trying to cut down on the way too many instances of implicit `any` types we have).

Reply:
I will say that typing this is getting kind of ugly. I have to cast the parent element, then cast the anchorElement back to `any` as the `automation` property does not actually exist on any type, it's just a custom added thing here.

```
const automation = ((item.parentElement as HaMenu)!.anchorElement as any)!.automation;
```

Is this going to be acceptable or need to clean it up somehow?

Reply:
I think it's fine for now.  The best goal would be to turn on `noImplicitAny` in tsconfig.  Casting as any is never great, but that's a different rule for another day.